spec/index.bs

```diff
@@ -3458,29 +3458,26 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
                                 [=exceeds the binding slot limits|exceed the binding slot limits=]
                                 of |this|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.
                             - For each {{GPUBindGroupLayoutEntry}} |entry| in |descriptor|.{{GPUBindGroupLayoutDescriptor/entries}}:
-                                - Let |bufferLayout| be |entry|.{{GPUBindGroupLayoutEntry/buffer}}
-                                - Let |samplerLayout| be |entry|.{{GPUBindGroupLayoutEntry/sampler}}
-                                - Let |textureLayout| be |entry|.{{GPUBindGroupLayoutEntry/texture}}
-                                - Let |storageTextureLayout| be |entry|.{{GPUBindGroupLayoutEntry/storageTexture}}
-
-                                - Exactly one of |bufferLayout|, |samplerLayout|, |textureLayout|,
-                                    or |storageTextureLayout| are not `undefined`.
-
+                                - Exactly one of |entry|["buffer"], |entry|["sampler"], |entry|["texture"], or
+                                    |entry|["storageTexture"] [=map/exists=].
+                                
                                 - If |entry|.{{GPUBindGroupLayoutEntry/visibility}} includes
                                     {{GPUShaderStage/VERTEX}}:
                                     - |entry|.{{GPUBindGroupLayoutEntry/buffer}}?.{{GPUBufferBindingLayout/type}}
                                         must not be {{GPUBufferBindingType/"storage"}}.
                                     - |entry|.{{GPUBindGroupLayoutEntry/storageTexture}}?.{{GPUStorageTextureBindingLayout/access}}
                                         must not be {{GPUStorageTextureAccess/"write-only"}}.
 
-                                - If the |textureLayout| is not `undefined` and
-                                    |textureLayout|.{{GPUTextureBindingLayout/multisampled}} is `true`:
+                                - If |entry|.{{GPUBindGroupLayoutEntry/texture}}?.{{GPUTextureBindingLayout/multisampled}}
```

Comment:
Speaking of which, the definition of `?.` in the "Dot syntax" section should also be updated to say something like "if Foo does not [=map/exist=] or is undefined or null"